src/components/tags/tags.js

```diff
@@ -5,7 +5,9 @@ import { ButtonMoreComponent } from "../button-more/button-more";
 import { TagDialogComponent } from "../tag-dialog/tag-dialog";
 
 import { TagListService } from "../../services/tag-service";
+import { AddTagDialogComponent } from "../add-tag-dialog/add-tag-dialog";
 import { TransactionListService } from "../../services/transaction-service";
+import { ConfirmDialogComponent } from "../confirm-dialog/confirm-dialog";
```

Comment:
move it after row 5 `import { TagDialogComponent } from "../tag-dialog/tag-dialog";`, don't mix services with components